logger.c

```diff
@@ -653,17 +653,18 @@ static void _logger_log_ext_write(logentry *e, item *it, uint8_t bucket) {
  * TODO: This and below should track and reprint the client fd.
```

Comment:
@dormando we can remove this TODO if it means this change.

Reply:
Yeah think so.